internal/fs/fs.go

```diff
@@ -1972,6 +1972,44 @@ func (fs *fileSystem) renameFile(
 	return nil
 }
 
+func (fs *fileSystem) releasePendingInodes(inodes *[]inode.DirInode) {
```

Comment:
rename this method to releaseInodes() -> you are passing inodes as argument. This method is not dealing with any pending inodes stuff.